Formula/emacs.rb

```diff
@@ -87,6 +88,14 @@ def install
     args << "--with-imagemagick" if build.with? "imagemagick"
     args << "--without-popmail" if build.with? "mailutils"
 
+    if build.with? "modules"
```

Comment:
You can probably just condense this into:
```ruby
args << "--with-modules" if !build.stable && build.with?("modules")
```
If we decide we're happy with this.